### PR TITLE
I've added Warming Stripes chart to the Examples page

### PR DESCRIPTION
**Pull request by avatorl**
I've added Warming Stripes chart to the Examples page ('Custom Designs' section)

### CODE REVIEW

**Comment by domoritz** [comment]
Nice. Do you think we could make this in Vega-Lite as well?

**Comment by avatorl** [comment]
So far I didn't even start learning Vega-Lite. I'm in love with Vega. And I already have many more good Vega examples https://powerofbi.org/deneb-vega-data-visualization-examples/ that i will want to pull one by one (after some additional tweaking and code cleaning).

**Comment by jheer** [comment]
Sorry for the long delay, but this example is now live on the Vega website. Thank you for submitting it!

File: docs/examples/warming-stripe.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+      "on": [
+        {"events": "@stripe:mouseover", "update": "datum.temp"},
+        {"events": "@stripe:mouseout", "update": "0"}
+      ]
+    }
+  ],
+  "data": [
+    {
+      "name": "dataset",
+      "url": "https://raw.githubusercontent.com/avatorl/Deneb-Vega/main/warming-stripe/global-temp.csv",
```
Comment (domoritz) [inline docs/examples/warming-stripe.json:44]: Should this be moved to vega datasets?

File: docs/examples/warming-stripe.json [hunk 2]
```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v5.json",
+  "description": "Warming Stripes Chart. Originally designed by Ed Hawkins.",
+  "usermeta": {
```
Comment (domoritz) [inline docs/examples/warming-stripe.json:4]: This is not something we have added to examples in the vega gallery. Maybe remove?

File: docs/examples/warming-stripe.json [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    "Original Idea URL": "https://www.climate-lab-book.ac.uk/2018/warming-stripes/",
+    "Read More URL": "https://en.wikipedia.org/wiki/Warming_stripes",
+    "Data Source URL": "https://climate.nasa.gov/vital-signs/global-temperature/"
+  },
+  "background": "white",
+  "width": 650,
+  "height": 200,
+  "padding": 10,
+  "autosize": "pad",
+  "config": {},
```
Comment (domoritz) [inline docs/examples/warming-stripe.json:18]: Remove this empty object

File: docs/examples/warming-stripe.json (hunk 1 above)
Comment (avatorl) [reply docs/examples/warming-stripe.json:44]: Seems to be an old file. It's `"url": "data/global-temp.csv",` in current version of Vega specification for this example (https://github.com/vega/vega/blob/main/docs/examples/warming-stripes.vg.json)

File: docs/examples/warming-stripe.json (hunk 3 above)
Comment (avatorl) [reply docs/examples/warming-stripe.json:18]: ok, I'll remove

File: docs/examples/warming-stripe.json (hunk 2 above)
Comment (avatorl) [reply docs/examples/warming-stripe.json:4]: ok

**Pull request merged**